src/AsciiDocProvider.ts

```diff
@@ -124,7 +124,7 @@ export default class AsciiDocProvider implements TextDocumentContentProvider {
         return new Promise<string>((resolve, reject) => {
             let text = doc.getText();
             let documentPath = path.dirname(doc.fileName);
-            let tmpobj = tmp.fileSync({ postfix: '.adoc', dir: documentPath });
+            let tmpobj = doc.isUntitled ? tmp.fileSync({ postfix: '.adoc' }) : tmp.fileSync({ postfix: '.adoc', dir: documentPath });
```

Comment:
The directory of `tmpobj` is `C:\Program Files\Microsoft VS Code` in Windows when `documentPath` is `"."`.

Reply:
Thanks for the fix :+1: